app/models/Workers.php

```diff
@@ -115,6 +115,22 @@ public function getEmail(): string
         return $this->email;
     }
 
+    /**
+     * Returns multidimensional array with id and full name of all workers
+     * @return array
+     */
+    public function getWorkersWithFullName()
```

Comment:
do you have workers without full name? I believe not....
This method seems to me that is get all workers and it orders the by first name, then last name. Name you methods accordingly, and also the php doc.